src/test/integration/index.ts

```diff
@@ -0,0 +1,5 @@
+import patchNamespace from './patchNamespace.js';
+
+console.log('Integration testing');
+
+patchNamespace();
```

Comment:
Should we await this? I'm assuming we would add more tests here in the future.

Reply:
yes, that was what i was thinking